### DISCUSSION TITLE
Badges

### DISCUSSION THREAD

**Discussion by loftwah** [EddieHubCommunity/LinkFree#6783] [Ideas]
Introduce a system of badges that users can earn for various activities, encouraging engagement and completion of profile elements.

**Comment by eddiejaoude** [comment]
This is a great idea 👍  do you have an example of badges?

**Comment by loftwah** [comment]
- Badge for 100% filled out profile
- Badge for using each of the features
- Badge for using all of the features
- Badge for being a premium user (at one point)
- Badge for giving a testimony
- Badge for receiving 5 testimonials
- Badge for visitors from x country
- Badge for having 1000/10k/100k views etc
- Badge for trying the different profile layouts
- Badge for being a user who came from using `json`

This is just a few and we could have heaps more based on the features of the app.

**Reply by Devanshukoli** [reply]
This will be certainly interesting.

**Comment by loftwah** [comment]
Would be similar to GitHub Achievements.

![Features](https://github.com/EddieHubCommunity/BioDrop/assets/19922556/7b16aaa8-4fc6-4d9e-b891-cca740c8dda1)
![Premium](https://github.com/EddieHubCommunity/BioDrop/assets/19922556/1a053ad6-641c-425a-b746-834f181ca57b)
![Master User](https://github.com/EddieHubCommunity/BioDrop/assets/19922556/92ee9ab1-ecef-440c-bade-ada6c122da16)
